packages/core/backend/sirius-components-core/src/main/java/org/eclipse/sirius/components/core/api/IRepresentationMetadataProvider.java

```diff
@@ -21,6 +23,8 @@
  * @author pcdavid
  */
 public interface IRepresentationMetadataProvider {
-    boolean canHandle(String representationId);
-    RepresentationMetadata handle(String representationId);
+    default boolean canHandle(String representationId) {
+        return this.handle(representationId).isPresent();
+    }
```

Comment:
We can just break this API and delete it entirely

Reply:
Done

Reply:
This was supposed to be temporary and I forgot to remove it.